.gitattributes

```diff
@@ -2,13 +2,10 @@ docs export-ignore
 examples export-ignore
 test export-ignore
 travis export-ignore
-.gitattributes export-ignore
-.gitignore export-ignore
-.travis.yml export-ignore
+/.* export-ignore
```

Comment:
It's probably a good idea to always ignore dotfiles, yeah, could you move that to the top with a comment?

Reply:
done